### ISSUE TITLE
MDExpress: Orthanc Configurations

### ISSUE THREAD

**Issue by vikashg** [Project-MONAI/monai-deploy#99]
I am trying one of the demos, when I send the images using terminal using `storescu` I can upload the images on Orthanc. However when I try to send the images from Orthanc to MONAI-DEPLOY I get the following error:
![image](https://user-images.githubusercontent.com/3863212/199346670-94d4e05c-b85a-4acb-b344-9d379b70f1a2.png)

I checked the Orthanc Configuration files and `RemoteAccessAllowed` is set to `true` and `AuthenticationEnabled` is set to `false`.
Any suggestions.

**Comment by mocsharp** [comment]
@vikashg You are correct, the `Insecure Setup` yellow warning displays when `RemoteAccessAllowed` is set to `true` and `AuthenticationEnabled` is set to false.  This was by design to make the experience easier.  

If you want to secure the Orthanc instance, you may update the configuration file and set `AuthenticationEnabled` to `true` and add `RegisteredUsers` as follows:

```
"RemoteAccessAllowed" : true,
"AuthenticationEnabled" : true,
"RegisteredUsers" : {
  "admin" : "password"
```

Please refer to [Orthanc ](https://book.orthanc-server.com/users/configuration.html) for additional configuration options.

**Comment by vikashg** [comment]
Hi @mocsharp 
So, I did what you said. and I think there is some AETitle mismatch 
![image](https://user-images.githubusercontent.com/3863212/199486498-e87bc90e-0c64-4531-9638-f0599f79c64c.png)

I am using the instructions for Liver Segmentation MAP.

**Comment by mocsharp** [comment]
@vikashg have you tried restarting the services by running
```
docker compose down
```
and then
```
docker compose up
```

If that doesn't work, please provide logs from Informatics Gateway by running

```
docker logs mdl-ig
```

**Comment by vikashg** [comment]
Seems like compose down and up solved that problem. However, it seems to take a long time due to insufficient storage. Atleast thats what I gather from logs. 

![image](https://user-images.githubusercontent.com/3863212/199817388-a5c2438c-85af-43bc-8b89-8b63bd909899.png)

**Comment by mocsharp** [comment]
@vikashg It looks like your drive is 95% filled; one option is to modify the [informatics-gateway.json](https://github.com/Project-MONAI/monai-deploy/blob/main/deploy/monai-deploy-express/configs/informatics-gateway.json) line 47, and set to `"watermarkPercent": 99,` and then restart all the services again.

**Comment by vikashg** [comment]
That is interesting because I just checked that our drives are not full
![image](https://user-images.githubusercontent.com/3863212/200029215-379e8f25-b5cd-49c5-b905-6c28f5f5d0d1.png)

I think if there is a way to clearly specify where this whole operation will go on, it will be helpful for testing.

**Comment by mocsharp** [comment]
@vikashg it's usually the drive that you are running MD express on.  
By default `$PWD/.md/mdig/payloads/` is mounted to `/payloads` in the Informatics Gateway container.
